src/Package/Impl/DataInspect/Commands/ImportDataSetTextFileCommand.cs

```diff
@@ -17,7 +17,10 @@ internal sealed class ImportDataSetTextFileCommand : SessionCommand {
         protected override void Handle() {
             base.Handle();
 
-            string filePath = VsAppShell.Current.BrowseForFileOpen(IntPtr.Zero, Package.Resources.CsvFileFilter);
+            string filePath = VsAppShell.Current.BrowseForFileOpen(
```

Comment:
Some code does but if it can accept in ctor it is better for testability. Sometimes it cant though.